.docker/docker-variables.env

```diff
@@ -20,3 +20,10 @@ LC_ALL=en_US.UTF-8
 PUSH_TO_CDASH=false
 
 XDG_RUNTIME_DIR=/tmp
+
+# HANA provider settings
+HANA_TESTS=true
+HANA_HOST=917df316-4e01-4a10-be54-eac1b6ab15fb.hana.prod-us10.hanacloud.ondemand.com
+HANA_PORT=443
+HANA_USER=QGIS_CI
+HANA_PASSWORD="3w8dkX:NDrs&"
```

Comment:
these lines would be better suited in .docker/docker-qgis-test.sh, no?

Reply:
I agree, it is done in the new change.